.pending/breaking/sdk/4706-Simplify-context

```diff
@@ -0,0 +1,11 @@
+4706 - Simplify context
+
+Replace complex Context construct with a simpler immutible struct.
+Only breaking change is not to support `Value` and `GetValue` as first class calls.
+We do embed ctx.Context() as a raw context.Context instead to be used as you see fit.
+
+Migration guide:
+`ctx = ctx.WithValue(contextKeyBadProposal, false)` ->
+`ctx = ctx.WithContext(context.WithValue(ctx.Context(), contextKeyBadProposal, false))`
```

Comment:
Are there any instances of these calls in the code? Seems like based on the diff this is a minimal change.

Reply:
in `x/gov/test_common.go` and `x/gov/end_blocker_test.go` to control failure mode or a test handler.